session/send.go

```diff
@@ -68,16 +75,57 @@ type Message struct {
 	IsDecoy bool
 }
 
+func (s *Session) WaitForSent(msgId *[cConstants.MessageIDLength]byte) error {
```

Comment:
would it look a bit cleaner to do something like this?
```
+type MessageID *[cConstants.MessageIDLength]byte
 type Message struct {
        // ID is the message identifier
-       ID *[cConstants.MessageIDLength]byte
+       ID MessageID
```

Reply:
Agreed, that is nicer.

Reply:
ok